### ISSUE TITLE
Play Space not resetting

### ISSUE THREAD

**Issue by BeMad4M3** [naelstrof/VRPlayspaceMover#27]
So, I got everything working, except for the playspace location not resenting to default, the only way ive gotten it to reset is by closing the playspace mover ap, and reopening it

**Comment by NotToBlame** [comment]
I also have this issue as well. My reset is bound to both grips but nothing is working. Also have to close and re-open to reset it

**Comment by naelstrof** [comment]
What's the command line arguments you two are using?

**Comment by BeMad4M3** [comment]
> What's the command line arguments you two are using?

the default one that comes with the program, and i try'd the command line, from Reetonzzz's youtube video, and that didnt work either

**Comment by NotToBlame** [comment]
I slightly edited the one I used but lost what it was I originally set due to re-install to see if it would fix but this is the original one which also doesn't reset
` -l 2 -r 2 --resetButtonMask 4 --physics --jumpMultiplier 130 --leftTogglePhysicsMask 4`

**Comment by naelstrof** [comment]
That should make it so if you press *both* grips on both vive wands, the playspace should reset, assuming you're using a Vive.

It's also important that your controllers are considered "left" and "right" controllers. Perhaps start PlayspaceMover with only controllers connected before you enable trackers to see if that helps.

**Comment by NotToBlame** [comment]
Yeah that's what I use is both grips. Though I disabled physics previously. I'll now try running with just controllers turn on and see if resetting counts. 
Side question: How do I find out if controllers are not considered "left" and "right"? or is just looking to see if the hand symbols come up enough?
